source/manual/howto-change-a-government-following-an-election.html.md

```diff
@@ -4,15 +4,14 @@ title: Change a government following a general election
 section: Publishing
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2019-07-12
+last_reviewed_on: 2019-12-05
 review_in: 6 months
 ---
 
 ## Change a government in Whitehall
 
-Following a general election or other reason for governmental change
-the current Government in Whitehall will need to be closed, and a new
-one created.
+Following a general election or other reason for governmental change the
```

Comment:
https://www.gov.uk/elections-in-the-uk is inconsistent about this, but should this be:

```suggestion
Following a General Election or other reason for governmental change the
```